registry.json

```diff
@@ -2821,5 +2821,32 @@
       "rpc": "https://exchainrpc.okex.org",
       "documentation": "https://okc-docs.readthedocs.io/en/latest"
     }
+  },
+  {
+    "id": "hedera",
+    "name": "Hedera",
+    "coinId": 3030,
+    "symbol": "HBAR",
+    "decimals": 8,
+    "blockchain": "Hedera",
+    "derivation": [
+      {
+        "path": "m/44'/3030'/0'/0'/0"
```

Comment:
why last `0` is not hardened?

Reply:
According to https://github.com/search?q=org%3Ahashgraph+3030&type=code it's unclear, however I've based the UT on the js sdk and the last one is not hardened